src/bp-search/classes/class-bp-search-posts.php

```diff
@@ -192,7 +192,7 @@ protected function generate_html( $template_type = '' ) {
 				}
 
 				// Add Boots Frond End Builder App.
-				if ( class_exists( 'ET_Builder_Plugin' ) ) {
+				if ( class_exists( 'ET_Builder_Plugin' ) && function_exists( 'et_fb_app_boot' ) ) {
```

Comment:
@yudhisthirnahar 
Also add this condition for below code
`if ( class_exists( 'ET_Builder_Plugin' ) ) {
   remove_filter( 'the_content', 'et_fb_app_boot', 1 );
}`

Reply:
@jitendrabanjara1991 to remove the filter it's not necessary to have the function et_fb_app_boot defined at that point of time, it will still remove, that's why I intentionally not added the condition

Reply:
@yudhisthirnahar 
Adding this check ensures that we handle potential errors gracefully. Since we cannot predict all scenarios where an error might occur, adding proper error handling will prevent unexpected issues during execution.